script.js

```diff
@@ -1,4 +1,52 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var computerThrew = getComputerSps();
+  console.log(`computer threw ${computerThrew}`);
+  console.log(`User threw ${input}`);
+  var myOutputValue = "You lost";
+
+  //input validation//
+  if (input != "scissors" || input != "paper" || input != "stone") {
+    myOutputValue = "Please key in scissors, paper or stone only.";
+  }
+
+  if (input == computerThrew) {
+    myOutputValue = "Its a draw";
+  }
+
+  if (input == "scissors" && computerThrew == "paper") {
+    myOutputValue = "You win";
+  }
+  if (input == "paper" && computerThrew == "stone") {
+    myOutputValue = "You win";
+  }
+  if (input == "stone" && computerThrew == "scissors") {
+    myOutputValue = "You win";
+  }
```

Comment:
Nice use of conditionals. Can simplify the logic here by using the || operator 
For example the win condition would look like this: 

if ( input == "scissors" && computerThrew== "paper" || 
     input == "paper" && input == "stone" || 
     input == "stone" && input == "scissors")  {
    // win   
 }